ts/src/xt.ts

```diff
@@ -955,6 +955,13 @@ export default class xt extends Exchange {
                     },
                 };
             }
+            const typeRaw = this.safeString (entry, 'type');
+            let type: Str = undefined;
+            if (typeRaw === 'FT') {
+                type = 'crypto';
+            } else {
```

Comment:
hum what is other?

Reply:
we have `other` kind in type. they can be whatever else (like NFT, POINTS, or somewhat other kind of rare products)

Reply:
@ttodua but aren't we missing this key from the type? 
![image](https://github.com/user-attachments/assets/aadbb43b-f98b-4242-a306-55db816e1d9c)

Reply:
wdym? we have `type?: str` there, and in tests we manually check the string to be among allowed `crypto, leverage, fiat, other` (these 4 strings)

Reply:
@ttodua sorry I was blind